godot-core/src/builtin/meta/mod.rs

```diff
@@ -7,27 +7,120 @@
 pub mod registration;
 
 mod class_name;
+mod godot_compat;
```

Comment:
I would name it `godot_compatible`, otherwise it sounds like a GDExtension compat module.